.travis.yml

```diff
@@ -48,6 +48,12 @@ matrix:
     - (cd diesel_codegen && cargo rustc --no-default-features --features "lint dotenv sqlite postgres mysql" -- -Zno-trans)
     - (cd diesel_infer_schema && cargo rustc --no-default-features --features "lint dotenv sqlite postgres mysql" -- -Zno-trans)
     - (cd diesel_compile_tests && travis-cargo test)
+  include:
+  - rust: nightly
+    env: RUSTFMT=YESPLEASE
```

Comment:
Why do we need this ?

Reply:
To make it clear what this run is.

Reply:
Oh in the travis UI, makes sense 👍

Reply:
Well we also need some env var that is different to make it a different matrix item than the normal nightly test runs.